hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/converter/TestFSQueueConverter.java

```
@@ -351,13 +351,16 @@ public void testAutoCreateV2FlagsInWeightMode() {
         csConfig.getBoolean(
             PREFIX + "root.misc.auto-queue-creation-v2.enabled", false));
 
+    //leaf queue root.admins.alice is removed from the below list
+    //adding reservation to a leaf, it's queueType changes to FSParentQueue
     Set<String> leafs = Sets.difference(ALL_QUEUES,
         Sets.newHashSet("root",
             "root.default",
```

"root.default" is misleading here, it should not have the auto-queue-creation-v2.enabled flag, however because of the Sets.difference call it's removed from this validation.